### PR TITLE
[#1501] Create Unit Tests for Feedback*QuestionDetails classes

### PR DESCRIPTION
**Pull request by aorlandou**
Part of #1501 

**Outline of Solution**

I added a few tests to cover the shouldChangesRequireResponseDeletion method for 2 classes:
- FeedbackMsqQuestionDetails
- FeedbackNumericalScaleQuestionDetails

### CODE REVIEW

**Review by cedricongjh — APPROVED** [review]
lgtm, thank you for your contribution to TEAMMATES

File: src/test/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetailsTest.java [hunk 1]
```diff
@@ -142,4 +142,52 @@ public void tesValidateQuestionDetails() {
...
         assertEquals(FeedbackNumericalScaleQuestionDetails.NUMSCALE_ERROR_STEP, errors.get(1));
     }
+
+    @Test
+    public void testShouldChangesRequireResponseDeletion_differentMinScale_shouldReturnTrue() {
+        FeedbackNumericalScaleQuestionDetails numDetails = new FeedbackNumericalScaleQuestionDetails();
+        numDetails.setMinScale(1);
+
+        FeedbackNumericalScaleQuestionDetails newNumDetails = new FeedbackNumericalScaleQuestionDetails();
+        numDetails.setMinScale(3);
```
Comment (EuniceSim142) [inline src/test/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetailsTest.java:152]: This should set min scale for `newNumDetails`. 
```suggestion
        newNumDetails.setMinScale(3);
```

File: src/test/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetailsTest.java [hunk 2]
```diff
@@ -142,4 +142,52 @@ public void tesValidateQuestionDetails() {
...
+        assertTrue(numDetails.shouldChangesRequireResponseDeletion(newNumDetails));
+    }
+
+    @Test
+    public void testShouldChangesRequireResponseDeletion_differentMaxScale_shouldReturnTrue() {
+        FeedbackNumericalScaleQuestionDetails numDetails = new FeedbackNumericalScaleQuestionDetails();
+        numDetails.setMaxScale(5);
+
+        FeedbackNumericalScaleQuestionDetails newNumDetails = new FeedbackNumericalScaleQuestionDetails();
+        numDetails.setMaxScale(2);
```
Comment (EuniceSim142) [inline src/test/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetailsTest.java:163]: Same thing here. Please also change all the other tests in this file.

**Review by EuniceSim142 — CHANGES_REQUESTED** [review]
Other than one change, lgtm!

**Comment by aorlandou** [comment]
Hey @EuniceSim142 thank you for the review you are correct. I made the necessary changes hopefully now it is ready to be merged. :grin:

**Review by EuniceSim142 — APPROVED** [review]
Lgtm, thanks for contributing to Teammates!

**Pull request merged**